creator-node/src/snapbackSM.js

```diff
@@ -1,18 +1,30 @@
 const config = require('./config')
 const Bull = require('bull')
+const axios = require('axios')
 const utils = require('./utils')
+const models = require('./models')
 const { logger } = require('./logging')
 
-const DevDelayInMS = 10000
+// For local dev, configure this to be the interval when SnapbackSM is fired
+const DevDelayInMS = 20000
+
+// Represents the maximum number of syncs that can be issued at once
+const MaxParallelSyncJobs = 10
+
+// Maximum number of time to wait for a sync operation
+const MaxSyncMonitoringDurationInMs = 36000
```

Comment:
all values are arbitrary for dev

Reply:
open to input prior to check-in but we need a hard cut-off for additional suggestions etc, going to set EOD monday

Reply:
assuming this runs every hour with 10 concurrent syncs at a time, 6 minutes means 60 syncs an hour if every sync takes 6 minutes. so that's a throughput of 1440 users a day if every one takes the max 6 mins which seems unlikely. most of our syncs take significantly less than that, not to mention this is only for users that have messed up state and this is about 15% of the users of cn1. so i think this is a reasonable max?

Reply:
fwiw this was actually 3600, not 360000ms so much smaller. going with 6 minutes for final check in